package/regenda/package

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+# Copyright (c) 2023 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(regenda)
+pkgdesc="Caldav agenda app"
+url=https://github.com/tenJirka/reGenda
+pkgver=0.0.1-1
+timestamp=2023-08-23T19:40Z
+section="utils"
+maintainer="tenJirka <jirka.steh@gmail.com>"
+license=GPL3
+makedepends=(build:python3-venv)
+installdepends=(python3 python3-lxml python3-pytz python3-yaml python3-requests python3-dateutil python3-six python3-urllib3 python3-idna)
+flags=(patch_rm2fb)
+source=(https://github.com/tenJirka/reGenda/archive/refs/tags/0.0.1.zip)
+sha256sums=(00d6a1e2f6b59f86ba47ea5c3444f24e112d24eb62082802653fe4649153da01)
+image=python
+
+build() {
+    python3 -m venv venv
+    source venv/bin/activate
+    mkdir "$srcdir"/python_modules
+    pip install caldav rm_pysas -t "$srcdir"/python_modules
+}
+
+package() {
+    install -D -m 644 -t "$pkgdir"/opt/etc/draft/ "$srcdir"/regenda.draft
+    install -D -m 644 -t "$pkgdir"/opt/etc/reGenda/ "$srcdir"/config.yml.example
+    install -D -m 644 "$srcdir"/icon.png "$pkgdir"/opt/etc/draft/icons/reGenda.png
+    install -D -m 755 -t "$pkgdir"/opt/reGenda/ "$srcdir"/run.py
+    install -D -m 644 -t "$pkgdir"/opt/reGenda/ "$srcdir"/languages.py
+    install -D -m 644 -t "$pkgdir"/opt/reGenda/ "$srcdir"/calendar_caldav.py
+    mkdir "$pkgdir"/opt/bin/
+    ln -s /opt/reGenda/run.py "$pkgdir"/opt/bin/reGenda
```

Comment:
Have you tested this package yet? I just realized that I'm not sure if this will work properly as a soft link, since the libraries you included are not in the same folder as the link, which is the main entrypoint.

I believe that python packages tend to create a separate python executable that will just run the main one's main function. For example:

```python
#!/usr/bin/python
# -*- coding: utf-8 -*-
import re
import sys
from remarkable_mouse.remarkable_mouse import main
if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])
    sys.exit(main())
```

Reply:
Yeah I did, works fine. This should not be necessary.
I was considering it as an option but when I find out it works well as it is, I decided to keep the simple solution.